File: java_client_ORM+REST/JavaSocketsFestival-ORM+REST-Final/Java11ChatSockets/ChatModel/src/main/java/festival/model/Persoana.java
package festival.model;

import java.io.Serializable;

public class Persoana implements Identifier<Long>, Serializable{
    private String nume;
    private String prenume;
    private Long id_persoana;

    public Persoana(Long id_persoana, String nume, String prenume) {
        this.id_persoana=id_persoana;
        this.nume = nume;
        this.prenume = prenume;
    }

    public Persoana() {
    }

    public Persoana(int idPersoana) {
    }

    public Persoana(String nume, String prenume) {
        this.nume=nume;
        this.prenume=prenume;
    }

    @Override
    public Long getId() {
        return id_persoana;
    }

    @Override
    public void setId(Long id_persoana) {
        this.id_persoana=id_persoana;

    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }

    public String getPrenume() {
        return prenume;
    }

    @Override
    public String toString() {
        return "Persoana{" +
                "nume='" + nume + '\'' +
                ", prenume='" + prenume + '\'' +
                '}';
    }
}

File: server_client_C_sharp/mpp_c_app1/FestivalModel/Persoana.cs
using System;

namespace FestivalModel
{
    [Serializable]

    public class Persoana : Identifier<long>
    {
        public String Nume { get; set; }
        public String Prenume { get; set; }
        public long IdPersoana { get; set; }

        public Persoana(long idPersoana, String nume, String prenume)
        {
            this.IdPersoana = idPersoana;
            this.Nume = nume;
            this.Prenume = prenume;
        }

        public Persoana(String nume, String prenume)
        {
            this.Nume = nume;
            this.Prenume = prenume;
        }

        protected Persoana()
        {
            
        }
    }
}
/*
@Override
public Long getId() {
    return id_p;
}

@Override
public void setId(Long aLong) {
    this.id_p=aLong;

}
*/
